docs/DeveloperGuide.adoc

```diff
@@ -316,30 +316,73 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Edit an item
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all items or only items of a specific `TYPE`
+2.  ResuMe shows a list of corresponding items
+3.  User checks for the `ID` of a specific item in the list to edit
+4.  User requests to edit a specific item in the list
```

Comment:
Maybe can combine these two lines? Borderline specific.
Maybe can change to User requests to edit a specific item based on ID?
Then extension can still follow